server/src/test/java/org/apache/druid/metadata/SQLMetadataSegmentManagerTest.java

```diff
@@ -123,10 +124,48 @@ public void testPoll()
         ImmutableList.of("wikipedia"),
         manager.getAllDataSourceNames()
     );
+    Assert.assertEquals(
+        ImmutableList.of("wikipedia"),
+        manager.getDataSources().stream().map(d -> d.getName()).collect(Collectors.toList())
+    );
     Assert.assertEquals(
         ImmutableSet.of(segment1, segment2),
         ImmutableSet.copyOf(manager.getDataSource("wikipedia").getSegments())
     );
+    Assert.assertEquals(
+                        ImmutableSet.of(segment1, segment2),
```

Comment:
Improper formatting

Reply:
Oh yeah, I should fix that. Sorry.

Reply:
Added in https://github.com/apache/incubator-druid/pull/7452.